lib/binding_rust/lib.rs

```diff
@@ -1318,6 +1400,15 @@ impl<'a> TreeCursor<'a> {
         };
     }
 
+    /// Move this cursor to the previous sibling of its current node.
+    ///
+    /// This returns `true` if the cursor successfully moved, and returns
+    /// `false` if there was no previous sibling node.
+    #[doc(alias = "ts_tree_cursor_goto_previous_sibling")]
```

Comment:
Could you add a note to this documentation that this method is slower than `goto_next_sibling` (because of the need to recalculate positions)?

Reply:
Also added a similar note to `ts_tree_cursor_goto_last_child`.